yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Don't check this file in.  There is no `package.json` at the root of this project.

Reply:
maybe we should chat about this one